{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -79,6 +79,9 @@
 {%- if cookiecutter.use_celery == 'y' %}
     "django_celery_beat",
 {%- endif %}
+{%- if cookiecutter.use_drf == "y" %}
+    "rest_framework.authtoken",
```

Comment:
I didn't realise that before, but surely `"rest_framework"` should also be included in this conditional.

We should also probably make it conditional in the requirements:

https://github.com/pydanny/cookiecutter-django/blob/5e16589d8edba2ad5ea97a427ac5f9ed5a3e0226/%7B%7Bcookiecutter.project_slug%7D%7D/requirements/base.txt#L32-L33

Would you mind moving the line above (78) into this and add a similar condition in the `requirements/base.txt` please?

Reply:
Sure!

Reply:
Just missing to move the `"rest_framework",` line above 😄 Thanks

Reply:
Alright. Will be checking this one as well :)

Reply:
Made the changes and merged this. Thanks!